ci/unit_tests/functions_delete_package/main_test.py

```diff
@@ -46,29 +46,29 @@ def setup_method(self):
 
     def teardown_method(self):
         """Delete my package, if it exists."""
-        existsResponse = self._getResponseFromPackage()
+        existsResponse = self._getResponseFromPackage(self.package)
         if existsResponse.status_code == 200:
             params = ['-c', os.path.join(self.dataBasePath, 'exampleFunctionsEmpty.cfg'),
-                '--cloudfunctions_package', self.package, '--cloudfunctions_namespace', self.namespace,
+                '--cloudfunctions_namespace', self.namespace,
                 '--cloudfunctions_url', self.cloudFunctionsUrl,
                 '--cloudfunctions_package', self.package,
                 '--cloudfunctions_apikey', self.apikey]
             self.t_noException([params])
 
-    def _getResponseFromPackage(self):
+    def _getResponseFromPackage(self, package):
```

Comment:
Why this is not the same as for `_checkPackageExists` and `_checkPackageDeleted`?

Reply:
Because the two functions use this function, they are not on the same level. So there is no reason to have the same args.